bots/images/scripts/fedora-atomic.bootstrap

```diff
@@ -10,4 +10,4 @@ BASE=$(dirname $0)
 # The Fedora URLs have the version twice in the name. for example:
 # https://dl.fedoraproject.org/pub/alt/atomic/stable/Fedora-Atomic-25-20160921.0/CloudImages/x86_64/images/Fedora-Cloud-Base-25-20160921.0.x86_64.qcow2
 $BASE/atomic.bootstrap "$1" "$url" \
-    "Fedora-Atomic-25-[0-9\.]+" "CloudImages" "x86_64" "images" "Fedora-Atomic-25-[0-9\.]+.x86_64.qcow2"
+    "Fedora-Atomic-[-0-9\.]+" "CloudImages" "x86_64" "images" "Fedora-Atomic-[-0-9\.]+.x86_64.qcow2"
```

Comment:
I agree. However this is exactly the same as any other updates needed to tests when the images change dramatically.

I think this is far preferable to the current status quo where we totally miss the point where these new images come online, and feedback on regressions and bugs in the operating system (such as this time around) is significantly delayed.